added additional args

ProFrenchToast opened the pull request:
Added the option to pass the additional args through to unity. This probably isn't the best solution but for now it should be fine.

### PR Description

On linux (and probably on other platforms) AAI doesnt not currently default to using the GPU for rendering visual observations making the environment run slower especially at higher timescales. Adding the option to pass your own commandline arguments can solve this and potentially other issues.

### Proposed change(s)

Takes the `additional_args` argument that is currently unused and appends it to the variable args before sending them as arguments to the unity executeable. Comments suggest that `additional_args` is no longer used for a specific reason however, I can't find that reason.

### Useful links (Github issues, discussion threads, etc.)

N/A

### Types of change(s)
- [ ] Bug fix
- [x] New feature
- [ ] Code refactor
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the changelog (if applicable)
- [ ] Updated the documentation (if applicable)
- [ ] Updated the migration guide (if applicable)

### Other comments (if any)

### Screenshots (if any)

ProFrenchToast committed:
added additional args

Added the option to pass the additional args through to unity. This probably isn't the best solution but for now it should be fine.

benaslater commented on animalai/environment.py:
```diff
@@ -153,7 +153,7 @@ def __init__(
             no_graphics=no_graphics,
             timeout_wait=self.timeout,
-            additional_args=args,
+            additional_args=args + (additional_args or []),
```
I think it would be good to print a "Use at your own risk" style warning when a user does these additional args, just to make sure it's visible e.g. in a year when someone is getting a weird failure because they copy/pasted some code that uses an additional arg they didn't want

ProFrenchToast committed:
added warnings to additional args

added some warnings to additional args to let users know that they are not well tested and im not sure why they were originally removed.
